arroyo-connectors/src/kafka.rs

```diff
@@ -95,6 +95,7 @@ impl Connector for KafkaConnector {
             table: serde_json::to_value(table).unwrap(),
             rate_limit: None,
             format: Some(format),
+            framing: schema.framing.clone(),
```

Comment:
Why do we need framing on both the OperatorConfig and the ConnectionSchema?

Reply:
The connection schema is what's used in the API and database, while the operator config is what is used to initialize the operator (i.e., what's passed from the compiler to the `from_config` method of the operators).